var/spack/repos/builtin/packages/imagemagick/package.py

```diff
@@ -29,6 +29,7 @@ class Imagemagick(AutotoolsPackage):
     depends_on('ghostscript')
     depends_on('ghostscript-fonts')
     depends_on('libsm')
+    depends_on('pkg-config', type='build')
```

Comment:
@healther note that `pkg-config` is a specific implementation of `pkgconfig`. Did you intend that this could only use that implementation?

Reply:
It could use any implementation. I'll fix this up in a bit

Reply:
Done